src/libs/actions/IOU.ts

```diff
@@ -6448,7 +6448,7 @@ function getReportFromHoldRequestsOnyxData(
         chatReport.reportID,
         chatReport.policyID ?? iouReport?.policyID ?? '',
         recipient.accountID ?? 1,
-        holdTransactions.reduce((acc, transaction) => acc + transaction.amount, 0) * (ReportUtils.isIOUReport(iouReport) ? 1 : -1),
+        ((iouReport?.total ?? 0) - ((iouReport?.unheldTotal ?? 0) + (iouReport?.nonReimbursableTotal ?? 0))) * (ReportUtils.isIOUReport(iouReport) ? 1 : -1),
```

Comment:
Since the `unheldTotal` doesn't include the `nonReimbursableTotal`, I add them together. If I have 3 expenses,

A: $1 (track, non-reimbursable)
B: $2 (held)
C: $3

The total is $6, unheldTotal is $3, and nonReimbursableTotal is $1.

held total = $6 - ($3 + $1) = $2

Reply:
I'm pretty confident that we can/should add them together because of this logic.
https://github.com/Expensify/App/blob/76c5c81542e598c147f57cf149122f4e124dfe10/src/libs/actions/IOU.ts#L6667-L6670

The logic above calculates the pay amount to show.

It subtracts the total from the nonReimbursableTotal. But if there are held expenses and it's paid partially, then we just use the unheldTotal which means the unheldTotal doesn't contain the nonReimbursableTotal.

unheldTotal = the total reimbursable amount that is not held